scaledbarcode.go

```diff
@@ -49,11 +49,16 @@ func (bc *intCSscaledBC) CheckSum() int {
 
 // Scale returns a resized barcode with the given width and height.
 func Scale(bc Barcode, width, height int) (Barcode, error) {
+	return ScaleWithFill(bc, width, height, color.White)
```

Comment:
I would suggest using the background of the used colorscheme instead of hardcoded white.

Reply:
To achieve this, I would need to request the ColorScheme via a parameter in the Scale function, which would be problematic due to the incompatibilities it generates.

I considered modifying the Barcode interface to add the ColorScheme() barcode.ColorScheme method, but I am unsure if it is appropriate to change this interface.

Reply:
You could add another interface to access the background. That way the original interface does not need to be changed. If the barcode is not implementing the new interface, you could then fallback to white